### PR TITLE
Clarify the purpose of starting.rst

### PR DESCRIPTION
**Pull request by stumblefiend**
We have several confusing pathways into multiple "getting started" types of content. Looking at the main software documentation landing page or knowing where to click from a search becomes confusing. I tried cleaning up this content and giving it a purpose, but there really looks to be only two valid scenarios the doc presents (the enterprise users section doesn't count because it links to the "interesting approaches" content instead of an actionable guide related to the topic of enterprise users.

Overall, this page may not be tangible in any meaningful way, so just wanted to hear other's thoughts...should we just delete or archive this content

### CODE REVIEW

File: docs/guide/starting.rst [hunk 1]
```diff
@@ -1,53 +1,28 @@
-============
-Starting off
-============
+=======================================================================
+Starting off: documentation best practices for developers
+=======================================================================
 
-So, you want to write the docs for your project?
+Want to write the docs for your project?
+You've come to the write [sic] place.
```
Comment (ericholscher) [inline docs/guide/starting.rst]: I think [sic] is mostly for text copied from elsewhere? Not 100% sure tho. 

```suggestion
You've come to the *write* place.
```

**Review by ericholscher — APPROVED** [review]
I think this is a useful "getting started" content, but could also be a top-level page in the guide for this content, instead of a subpage? 🤷

File: docs/guide/starting.rst (hunk 1 above)
Comment (plaindocs) [reply docs/guide/starting.rst]: Right, a kinda "we know this is wrong, but didn't fix it cos it's a direct quote.

**Comment by stumblefiend** [comment]
I think this PR is good enough for now and should be an improvement over what was there originally.

**Pull request merged**